lib/sidekiq/cloudwatchmetrics.rb

```diff
@@ -4,7 +4,7 @@
 require "sidekiq/api"
 require "sidekiq/util"
 
-require "aws-sdk-cloudwatch"
+require "aws-sdk"
```

Comment:
Can fall back on old version

```suggestion
begin
  rescue "aws-sdk-cloudwatch"
rescue LoadError
  require "aws-sdk"
end
```